v2/README.md

```diff
@@ -0,0 +1,50 @@
+#  go-spiffe (v2)
+
+This library is a convenient Go library for working with [SPIFFE](https://spiffe.io/).
+
+It leverages the 
```

Comment:
nit: we may reword this removing the `And more...` bullet. Something like
It leverages the [SPIFFE Workload API](https://github.com/spiffe/spiffe/blob/master/standards/SPIFFE_Workload_API.md), providing high level functionality that includes:

Reply:
:+1: